app/containers/Settings/Settings.jsx

```diff
@@ -18,6 +19,8 @@ type Props = {
   explorer: string,
   setCurrency: string => any,
   currency: string,
+  setTheme: string => any,
+  theme: string,
```

Comment:
Would be nice to have theme type

Reply:
A theme type would simply be a restricted set of strings. In this case it's currently `Light` or `Dark`. I think we should avoid defining that list over and over again in multiple places, but if you think it adds value here I'll add it.